NangPaGo-be/src/test/java/com/mars/NangPaGo/domain/community/service/CommunityLikeServiceTest.java

```diff
@@ -0,0 +1,158 @@
+package com.mars.NangPaGo.domain.community.service;
+
+import com.mars.NangPaGo.domain.community.dto.CommunityLikeResponseDto;
+import com.mars.NangPaGo.domain.community.entity.Community;
+import com.mars.NangPaGo.domain.community.entity.CommunityLike;
+import com.mars.NangPaGo.domain.community.repository.CommunityLikeRepository;
+import com.mars.NangPaGo.domain.community.repository.CommunityRepository;
+import com.mars.NangPaGo.domain.user.entity.User;
+import com.mars.NangPaGo.domain.user.repository.UserRepository;
+import com.mars.NangPaGo.support.IntegrationTestSupport;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class CommunityLikeServiceTest extends IntegrationTestSupport {
+
+    @Autowired
+    private CommunityLikeService communityLikeService;
+
+    @Autowired
+    private CommunityLikeRepository communityLikeRepository;
+
+    @Autowired
+    private CommunityRepository communityRepository;
+
+    @Autowired
+    private UserRepository userRepository;
```

Comment:
컨벤션: 테스트할 Service 코드를 맨 아래에 작성해주세요. Bean이 생성되는(필요한) 순서대로 작성하면 코드를 이해하기 쉬워져요.

```suggestion
    @Autowired
    private CommunityLikeRepository communityLikeRepository;
    @Autowired
    private CommunityRepository communityRepository;
    @Autowired
    private UserRepository userRepository;
    
    @Autowired
    private CommunityLikeService communityLikeService;
```